Dockerfile

```diff
@@ -16,6 +16,8 @@
 # under the License.
 #
 
+# TODO: minimize the docker image size, now 524MB !!!
+
```

Comment:
Haha, why so huge?

Reply:
Didn't notice this before because I used alpine as base image, maybe because I use `apk` to download some dependencies inside the image, need sometime to investigate and minimize it

Reply:
After compressed, it's 184MB, https://hub.docker.com/repository/docker/apache/skywalking-eyes/tags?page=1&ordering=last_updated